src/gsym.rs

```diff
@@ -71,6 +74,75 @@ impl SymResolver for GsymResolver {
         None
     }
 
+    fn find_line_info(&self, addr: u64) -> Option<AddressLineInfo> {
+        let addr = addr - self.loaded_address;
+        let idx = find_address(&self.ctx, addr);
+        let symaddr = self.ctx.addr_at(idx);
+        if addr < symaddr {
+            return None;
+        }
+        let addrinfo = self.ctx.addr_info(idx);
+        if addr >= (symaddr + addrinfo.size as u64) {
+            return None;
+        }
+
+        let addrdatas = parse_address_data(addrinfo.data);
+        for addrdata in addrdatas {
+            if addrdata.typ != InfoTypeLineTableInfo {
+                continue;
+            }
+            let lthdr = parse_line_table_header(addrdata.data);
+            if lthdr.is_err() {
+                #[cfg(debug_assertions)]
+                eprintln!("invalid line table header");
```

Comment:
Again, a library probably shouldn't emit strings without context to stderr. Maybe that's fine-ish here because of `debug_assertions` cfg, but then it seems as if this may be relevant information for users to understand/debug behavior? So perhaps use `log` instead?